sashimi-webapp/src/logic/processor/processorPump.js

```diff
@@ -0,0 +1,41 @@
+// Importing modules
+const cm = require("../contentManipulator/contentManipulator");
```

Comment:
Using `cm`, `cp`, `mdp` etc may not be a good software engineering practice.  
Consider using the full name instead
- `cm` -> `contentManipulator`
- `cp` -> `conditionalProcessor`
- ...

or trim the filename of the library.